repos/c2sm/packages/icon/package.py

```diff
@@ -35,6 +35,10 @@ class Icon(SpackIcon):
     maintainers('jonasjucker', 'huppd')
 
     version('develop', submodules=True)
+    version("2024.10",
+            tag="icon-2024.10",
+            git='git@gitlab.dkrz.de:icon/icon.git',
```

Comment:
What about adding official open-source releases here? E.g. `https://gitlab.dkrz.de/icon/icon-model/-/tree/release-2024.07-public?ref_type=heads`

Reply:
These versions are defined in the parent recipe:
https://github.com/spack/spack/blob/f25cbb0fe48acb13696ab2a211c6a889333e9d71/var/spack/repos/builtin/packages/icon/package.py#L22

I think we should leave this there.
Sergey will take care of that. And through inheritance we get the version ultimately.